tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py

```diff
@@ -18,7 +18,7 @@
 from wazuh_testing.tools.remoted_sim import RemotedSimulator
 from wazuh_testing.tools.services import control_service
 
-from conftest import AR_LOG_FILE_PATH, wait_received_message_line, wait_start_message_line, \
+from .conftest import AR_LOG_FILE_PATH, wait_received_message_line, wait_start_message_line, \
```

Comment:
Why are you changing the import here?

Reply:
Removed changes in imports in 618a8892f93dd794a6000baa838503371ec1eb6d